terraform/domain.tf

```diff
@@ -33,3 +33,14 @@ resource "aws_route53_record" "www" {
   ttl     = "300"
   records = ["dandi.github.io"]
 }
+
+resource "aws_route53_record" "email" {
+  zone_id = aws_route53_zone.dandi.zone_id
+  name    = "email"
```

Comment:
With this `name`, you're creating an MX record for anything sent `@email.dandiarchive.org`, which will not cover anything `@dandiarchive.org`. I'm not sure if that's intended.

Reply:
Oops! So it should actually be `""`, is that right?

Reply:
Yeah, I find it helpful to put:
```
name = "" # apex
```
to clarify the intent.